apps/backend/src/datasources/postgres/CallDataSource.ts

```diff
@@ -44,6 +44,11 @@ export default class PostgresCallDataSource implements CallDataSource {
 
   async getCalls(filter?: CallsFilter): Promise<Call[]> {
     const query = database('call').select(['*']);
+
+    if (filter?.shortCode) {
+      query.where('call_short_code', filter.shortCode);
+    }
```

Comment:
Do we need it to be exact or we can benefit from making match with a like.

Reply:
Great point, thanks.